cms/templatetags/cms_js_tags.py

```diff
@@ -45,10 +45,20 @@ def render_placeholder_toolbar_js(placeholder, render_language, content_renderer
     )
 
     def _render_plugin_js(plugin):
+        if plugin.plugin_type in plugin_children:
```

Comment:
please use a try/except instead of an if for both:

```python
try:
    child_classes = plugin_children[plugin.plugin_type]
except KeyError:
    # Plugin has opted out of the cached child classes
    # cache_child_classes is set to False
    child_classes = plugin.plugin_class.get_child_classes(slot=slot, page=page, instance=plugin)
```